docs/tools/built-in-tools.md

```diff
@@ -170,17 +171,17 @@ to use built-in tools with other tools by using multiple agents:
     search_agent = Agent(
         model='gemini-2.0-flash',
         name='SearchAgent',
-        instruction="""
+        instruction='''
```

Comment:
while ''' and """ is syntactically equivalent, this code should not be changed (8 instances)